modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/files/Read.java

```diff
@@ -62,7 +62,7 @@
     @Override
     public BValue[] execute(Context context) {
         BStruct file = (BStruct) getRefArgument(context, 0);
-        int bytesToRead = getIntArgument(context, 0);
+        int bytesToRead = (int) getIntArgument(context, 0);
```

Comment:
This value is used for BufferedInputStream:read [1] where bytes to read is an int.

[1] https://docs.oracle.com/javase/7/docs/api/java/io/BufferedInputStream.html#read%28byte[],%20int,%20int%29